boards/makerbase/mks_canable_v20/support/openocd.cfg

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2024 Alexander Kozhinov <ak.alexander.kozhinov@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+proc version_compare {v1 v2} {
+	foreach n1 [split $v1 .] n2 [split $v2 .] {
+		if {$n1 < $n2} {return -1}
+		if {$n1 > $n2} {return 1}
+	}
+	return 0
+}
+
+set current_version [version]
+regexp {([0-9]+\.[0-9]+\.[0-9]+)} $current_version -> current_version
+
+if {[version_compare $current_version "0.12.0"] >= 0} {
+	# OpenOCD version is 0.12.0 or newer
+	set transport_type dapdirect_swd
+} else {
+	# OpenOCD version is older than 0.12.0
+	set transport_type hla_swd
+}
+
+source [find interface/stlink.cfg]
+source [find target/stm32g4x.cfg]
+
+transport select $transport_type
+
+if {[version_compare $current_version "0.12.0"] >= 0} {
+	# OpenOCD version is 0.12.0 or newer
+	init
+	rtt setup 0x20000000 10000 "SEGGER RTT"
+	rtt start
+	rtt server start 9090 0
+} else {
+	# OpenOCD version is older than 0.12.0
+	echo "Warn: Segger RTT is not supported for this OpenOCD version: $current_version"
+}
```

Comment:
This looks very generic and should probably not just be set for one board. It could go into a common OpenOCD configuration file.

Reply:
Do you mean by `common` the configuration file delivered with OpenOCD itself?

Reply:
I meant either `boards/common/openocd.board.cmake`. I would suggest submitting this in a separate PR.

Reply:
> I meant `boards/common/openocd.board.cmake`. I would suggest submitting this in a separate PR.

It is a good advice. Since it wil blow up the current scope and I can not test it for other boards  - I would prefer to leave it here first. After merging this PR I would reallocate it to boards/common/openocd.board.cmake file. Or you can fell free to do so. I can support.

Reply:
I've dropped it and will prepare separate PR.

Reply:
Segger RTT section is moved to more general approach within the PR #82413